data/tutorials/platform/bp_03_run_executables_and_tests.md

```diff
@@ -39,6 +39,14 @@ For instance, if you've put your `dune` file in `bin/dune` with the following co
 
 You can run it with `dune exec bin/main.exe` or `dune exec my-app`.
 
+## Building the Project When Files Changes
+
+The `dune exec <executable_path>.exe` could take some time to compile a project when multiple files are involved. It could be interesting to have a process that recompiles each file as soon as they are changed. Then after the last saved file, it is possible that it remains only a last file to compile or just the linking process of all files and libraries.
+
+The command `dune build --watch` looks for things that need to be compiled, and afterwards, just waits for a modified file that will trigger the compilation of needed modules.
+
+However, `dune` locks the build directory, then it is not possible to launch two `dune` commands at the same time. The `dune build --watch` has to be stopped (typing Ctrl-C) before launching the application. Or we can launch the application without the help of Dune by typing `_build\default\<executable_path>.exe`.
```

Comment:
A remark about launching `dune build --watch` in the background should be added. For clarity, I believe an item list with the various cases should be added too.

Reply:
Ok for the background use of `dune build --watch`, but since this command prints the evolution of the compilation, I find it better to launch it on a dedicated terminal... and work with other terminals.

Do you mean a summary with the different `dune` commands. `dune exec ...`, `dune build --watch`, `dune test`. There are not a lot of cases.